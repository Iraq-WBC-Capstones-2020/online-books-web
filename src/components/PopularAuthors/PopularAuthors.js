import React from 'react';
import './style.scss';
import { MDBContainer, MDBBtn, MDBRow, MDBCol } from 'mdbreact';
import { useTranslation } from 'react-i18next';

function Author() {
  const { t } = useTranslation();
  const popularAuthor = [
    'Zen Cho',
    'John Doe',
    'Ahemd Doe',
    'Ali Doe',
    'Alif Doe',
    'john Doe',
    'rema bedr',
    'Ali Doe',
    'Alif Doe',
    'john Doe',
    'Ali Doe',
    'John Doe',
  ]; 
  return (
    <div>
      <MDBContainer className="mt-5">
        <div>
          <h1 className="text-center text-3xl">{t('mainPage.PopularAuthors')}</h1>
        </div>
        <MDBRow className="mt-5">
          {popularAuthor.map((author) => {
            return (
              <MDBCol
                key={author}
                size="12"
                sm="2"
                md="3"
                className="mt-4 d-flex justify-content-center align-items-center"
              >
                <MDBBtn color="light-blue" className="px-2 rounded-pill button">
                  {author}
                </MDBBtn>
              </MDBCol>
            );
          })}
        </MDBRow>
      </MDBContainer>
    </div>
  );
}

export default Author;
