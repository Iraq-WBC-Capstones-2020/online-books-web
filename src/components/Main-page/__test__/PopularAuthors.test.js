import React from 'react';
import renderer from 'react-test-renderer';
import PopularAuthors from './../PopularAuthors';

test(' PopularAuthors', () => {
  const tree = renderer.create(<PopularAuthors />).toJSON();
  expect(tree).toMatchSnapshot();
});
