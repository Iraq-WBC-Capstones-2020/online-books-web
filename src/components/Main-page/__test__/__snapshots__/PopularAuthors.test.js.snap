// Jest Snapshot v1, https://goo.gl/fbAQLP
exports[` PopularAuthors snapshot 1`] = `
   <div  className=" container mt-5"
   data-test="container">
        <div>
          <h1 className="text-center text-3xl">Popular authors</h1>
        </div>
        <div className="mt-5" data-test="row">
          {popularAuthor.map((author) => {
            return (
              <div
                key={author}
               data-test="col"
                className=" col-12 col-sm-2 col-md-3 mt-4 d-flex justify-content-center align-items-center"
              >
                <button color="light-blue" className="px-2 rounded-pill button">
                  {author}
                </button>
              </div>
            );
          })}
        </div>
      </div>
`;
